packages/map-template/src/components/Wayfinding/Wayfinding.scss

```diff
@@ -26,8 +26,20 @@
         position: absolute;
         top: var(--spacing-x-small);
         right: var(--spacing-x-small);
-        background: none;
-        border: none;
+        background-color: var(--tailwind-colors-gray-100);
+        border: 1px solid var(--tailwind-colors-gray-200);
+        margin-left: auto;
+        width: var(--spacing-xx-large);
+        height: var(--spacing-xx-large);
+        border-radius: var(--rounding-large);
+        display: grid;
+        justify-content: center;
```

Comment:
`place-items`?

Reply:
`place-items` is a constituent for `align-items` and `justify-items`, but in this case we have `justify-content`, so that won't work.